index.html

```diff
@@ -290,7 +298,10 @@ <h3 id="File_Names" style="left: -46px; position: relative;">File Names</h3>
           <code>foo_bar.hpp</code> and
           <code>foo_bar.cpp</code>, defining a class called
           <code>FooBar</code>.</p>
-
+          
+          <p>There is an exemption to this rule when writing tests. In this case tests should have .test.cpp as 
```

Comment:
Why not write "This rule may not apply in testing. For more information see `<link here>`"? And then add the file naming to the test section? I think that would be a bit more concise.

Reply:
Yes that's a good suggestion 😄